runtime/float.go

```diff
@@ -28,12 +29,16 @@ var FloatType = newBasisType("float", reflect.TypeOf(Float{}), toFloatUnsafe, Ob
 // Float represents Python 'float' objects.
 type Float struct {
 	Object
-	value float64
+	value    float64
+	hashOnce sync.Once
+	hash     int
 }
 
 // NewFloat returns a new Float holding the given floating point value.
 func NewFloat(value float64) *Float {
-	return &Float{Object{typ: FloatType}, value}
+	result := Float{Object: Object{typ: FloatType}}
```

Comment:
I assume this is changed because of the new struct member. What about using named field assignment like this:

```
return &Float{Object: Object{typ: FloatType}, value: value}
```